pipelines/A23/queries.log

```diff
@@ -60,8 +61,8 @@ SELECT T1.name, T1.preisanderung_chf FROM (     SELECT         S.name,         (
 Query Nr. 5
 -- Wie viele Wohnbauländer wurden 2018 in Basel-Landschaft gekauft?
 SELECT SUM(T.falle) FROM basel_land_quadratmeterpreis_wohnbauland_nach_gemeinde_und_jahr AS T WHERE T.jahr=2018;
- SUM(T.falle)
-          220
+ sum
```

Comment:
Please find a more specific header for the output column such as `anzahl_wohnbaulaender` and add context `jahr` and `Canton`. The spatial_unit table join is missing on this query. We should for consistency reasons always have a spatial join.